errors/errors.go

```diff
@@ -18,36 +18,35 @@ package errors
 
 import (
 	"fmt"
-	"net/http"
 )
 
+// Reason is an enumeration of possible failure causes. Each Reason
+// must map to a format which is a string containing ${formatArgu1}.
+// Exp:
+// Reason "kind:NotFound" may map to Format "${kindName} was not found".
+// Reason "Status:Sleep" may map to Format "${Name} is sleeping now"
+type Reason string
+
+// Error is the basic form of an error returned by nirvana.
 type Error interface {
```

Comment:
The Error is confused with the built-in Error interface.
The similar concept in k8s is called [APIStatus](https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go#L45:1)